### DISCUSSION TITLE
Build for Production

### DISCUSSION THREAD

**Discussion by M3rcena** [tolgee/tolgee-platform#2359] [Q&A]
Hello, how can we build the source code for production mode?

**Comment by JanCizmar** [comment]
Hello! 

May I ask would you like to do it? There are public binaries distributed via [DockerHub](https://hub.docker.com/r/tolgee/tolgee) or via [GitHub releases](https://github.com/tolgee/tolgee-platform/releases).

**Reply by M3rcena** [reply]
Well we want to add more pages such as translations information and other stuff for our translation teams while we have some mandatory rules about symbols such as `>` and `>>>` which should have  `\n`  and many more info. So we had to edit the pages add new pages and now we need to build it for production.

**Reply by JanCizmar** [reply]
Hey!

To build the production docker image, you can just run `./gradlew docker`. Then you can run it or push it to your docker repository. 

However, please assure you either remove all the `ee` code or you purchase the ee license for your self-hosted instance. Tolgee has 10 user limit for free-self hosted and "stuff for our translation teams" suggests that your instance might be used by more then 10 users. Using Tolgee with `ee` features without respecting the free limits is a license breach.

If you're school or open-source project, or NGO organization, we are able to provide you the license for free of with major discount.

**Reply by M3rcena** [reply]
Hi @JanCizmar 

Actually we are a small group of students which started a discord bot project and we are looking forward to making the discord bot translatable for every language.

And we already had many expenses due to hosting and everything else so we are limited to other resources while we don't have a job or anything.

Is there anything we can do for the licence?

**Reply by JanCizmar** [reply]
Please DM me on https://tolg.ee/slack

**Reply by M3rcena** [reply]
Wonderful! 

I just sent you a message there!
